services/app/src/components/avatarPicture/GhostAvatar.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import { Avatar } from 'antd';
+import { getAvatarPxSize, getAvatarFontSize } from '../../helpers/getAvatarSizes';
+
+const GhostAvatar = ({ avatarText, size }) => (
+  <Avatar 
```

Comment:
Make a wrapper Avatar component, that would wrap antd's Avatar and apply the styling inside the wrapper to avoid copy-pasting the same thing.